deployment/kustomizations/README.md

```diff
@@ -0,0 +1,86 @@
+# Deployment of WrenAI to k8s with kustomization
+1. Make sure you sutisfy the dependancies we are using to deploy WrenAI.
+2. Adjust the values and manifests accordingly to your k8s environment.
+3. Deploy Secrets separetly.
+4. Then deploy the inflated kustomized app.
+
+Note: without authentication, once you publish this in the internet enyone can acess your app, see your data and modify your settings!
+## Dependancies used in this kustomization:
+- nginx.ingress
+- external-dns
+- cert-manager
+- kubectl kustomize
+- helm (for minikube)
+
+## Steps to deploy:
+
+- Before you deploy check out manifests in the `deployment/kustomizations` folder and modify them you your k8s environment.
+The `deployment/kustomizations` folder contains a kustomization.yaml file that will inflate the manifests in to a `deployment/kustomizations/wrenai.kustimized.yaml` file used to deploy the app to your k8s.
```

Comment:
```suggestion
```suggestion
Before deploying, check out the manifests in the `deployment/kustomizations ` folder and modify them for your Kubernetes environment.

The `deployment/kustomizations` folder contains a `kustomization.yaml` file that will inflate the manifests into a `deployment/kustomizations/wrenai.kustomized.yaml` file used to deploy the app to your Kubernetes cluster.
```
```

Reply:
I think you forget the closing 3 symbols for the suggestion block at the end. @wwwy3y3

Reply:
ohh, good catch.